ambry-clustermap/src/main/java/com/github/ambry/clustermap/TaskUtils.java

```diff
@@ -38,4 +41,28 @@ protected static Pair<String, Integer> getHostNameAndPort(String instanceName, i
     }
     return new Pair<>(hostname, port);
   }
+
+  protected static boolean removeConfig(Set<String> config){
```

Comment:
I have made methods name generic. 
By using collection also I can't make these two methods into one, since I need to use Map in another method. I think we can keep it like this for now. If we find any need then we can think of this.